File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        map<int,int> mp;
        for(int i = 0; i < nums.size(); i++){
            mp[nums[i]]++;
        }
        k = nums.size() - k;
        for(auto it: mp){
            if(it.second > k) return it.first;
            k -= it.second;
        }
        return -1;
    }
};

File: Data Structures & Algorithms/sort-an-array/submission-0.py
class Solution:
    def sortArray(self, nums: List[int]) -> List[int]:
        pq = []
        for num in nums:
            heapq.heappush(pq, num)

        ans = []
        while pq:
            num = heapq.heappop(pq)
            ans.append(num)
        return ans